### PR TITLE
Router user,donation & businessRules user func upd

### PR DESCRIPTION
**Pull request by VladimirKarin**
Minor updates at:
 - Router:
   - users (get. method update)
   - donations (get. method update)
 - BusinessRules:
   - users (new func) 
 - index:
   - comments style update.

### CODE REVIEW

File: routes/donations.js [hunk 1]
```diff
@@ -9,11 +9,19 @@ const {
...
+        res.status(404).send(error.message);
+    }
 });
 
 router.get('/total', (req, res) => {
-    res.status(200).json(getTotalSumDonatedForIdea(req.query.ideaId));
+    try {
+        res.status(200).json(getTotalSumDonatedForIdea(req.query.ideaId));
+    } catch (error) {
+        res.status(404).send(error.message);
```
Comment (fedosejev) [inline routes/donations.js:23]: If unexpected server error happens, then server should return HTTP 500, not 404

File: routes/users.js [hunk 2]
```diff
@@ -2,11 +2,27 @@ const express = require('express');
...
+        } else {
+            res.status(403).send(
+                'Access Forbidden. Only Admin has the right to see the users list.'
+            );
+        }
+    } else {
+        res.status(403).send(
+            'Access Forbidden. Only Admin has the right to see the users list.'
+        );
+    }
```
Comment (fedosejev) [inline routes/users.js:25]: This is repeated twice:
```js
res.status(403).send(
    'Access Forbidden. Only Admin has the right to see the users list.'
);
```

Can you simplify the code above to not repeat it?

File: routes/donations.js [hunk 3]
```diff
@@ -9,11 +9,19 @@ const {
 const { getUser } = require('../models/users');
 
 router.get('/', (req, res) => {
-    res.status(200).json(getDonationsForIdea(req.query.ideaId));
+    try {
+        res.status(200).json(getDonationsForIdea(req.query.ideaId));
+    } catch (error) {
+        res.status(404).send(error.message);
```
[the remark this answers is not in the document]
Comment (VladimirKarin) [reply routes/donations.js]: Phind fixed.

File: routes/donations.js (hunk 1 above)
Comment (VladimirKarin) [reply routes/donations.js]: Phind fixed.

File: routes/users.js (hunk 2 above)
Comment (VladimirKarin) [reply routes/users.js:28]: Phind fixed it.

**Review by fedosejev — APPROVED** [review]

**Pull request merged**